### PR TITLE
add release notes for 0.2.0

### PR DESCRIPTION
**Pull request by pmeier**

### CODE REVIEW

File: docs/references/release-notes.md [hunk 1]
```diff
@@ -14,12 +189,50 @@
 ### New Contributors
 
 - [@peachkeel](https://github.com/peachkeel) made their first contribution in
-  [#184](https://github.com/Quansight/ragna/pull/244)
+  [#244](https://github.com/Quansight/ragna/pull/244)
```
Comment (pmeier) [inline docs/references/release-notes.md:192]: Driveby fix

File: docs/references/release-notes.md [hunk 2]
```diff
@@ -14,12 +189,50 @@
...
 - [@nenb](https://github.com/nenb) made their first contribution in
   [#252](https://github.com/Quansight/ragna/pull/252)
 
 ## Version 0.1.2
 
+### Breaking changes
```
Comment (pmeier) [inline docs/references/release-notes.md:198]: While writing the breaking changes for `0.2.0`, I realized this change was never documented. I had it written up anyway, so I put it here. I would also be fine to remove this again to avoid "changing history".

File: docs/references/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,298 @@
 # Release notes
 
+## Version 0.2.0
+
+### Feature changes and enhancements
+
+- Ragnas `0.1.x` releases were build on a task queue backend. This turned out to be a
```
Comment (pavithraes) [inline docs/references/release-notes.md:7]:
```suggestion
- Ragna's `0.1.x` releases were built on a task queue backend. This turned out to be a
```

**Review by pavithraes — APPROVED** [review]
Thanks for documenting a clear set of changes, especially the config file changes!

I noticed a couple of typos, which I'll commit and we can merge. :)
